lib/application-processing/resolvers.ts

```diff
@@ -885,12 +889,19 @@ export const updateApplicationProcessingGenerateInvoice: Resolver<
   );
 
   // Upload pdf to s3
-  const s3InvoiceKey = `invoices/${randomUUID()}/${invoice.invoiceNumber}.pdf`;
+  // file name format: PP-Receipt-P<YYYYMMDD>-<invoice number>.pdf
+  const fileName = `PP-Receipt-P${formatDateYYYYMMDD(invoice.createdAt).split('-').join('')}-${
+    invoice.invoiceNumber
+  }.pdf`;
```

Comment:
Need to confirm that this will not add any extra whitespace characters when saving

Reply:
replaced this so the linter doesn't format it like that

```javascript
const createdAtYYYMMDD = formatDateYYYYMMDD(invoice.createdAt).replace(/-/g, '');
const fileName = `PP-Receipt-P${createdAtYYYMMDD}-${invoice.invoiceNumber}.pdf`;
const s3InvoiceKey = `rcd/invoices/${fileName}`;
```